src/main/java/seedu/sellsavvy/logic/commands/ordercommands/DeleteOrderCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.sellsavvy.logic.commands.ordercommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.sellsavvy.commons.util.CollectionUtil.requireAllNonNull;
+
+import javafx.collections.ObservableList;
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.commons.util.ToStringBuilder;
+import seedu.sellsavvy.logic.Messages;
+import seedu.sellsavvy.logic.commands.Command;
+import seedu.sellsavvy.logic.commands.CommandResult;
+import seedu.sellsavvy.logic.commands.exceptions.CommandException;
+import seedu.sellsavvy.model.Model;
+import seedu.sellsavvy.model.order.Order;
+import seedu.sellsavvy.model.order.OrderList;
+
+/**
+ * Deletes an order under a specified person.
+ */
+public class DeleteOrderCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteOrder";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes an order under the specified person. "
+            + "Parameters: INDEX (must be a positive integer) "
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_ORDER_SUCCESS = "Deleted Order: %1$s";
+
+    private final Index index;
+
+    /**
+     * Creates an DeleteOrderCommand to delete the specified {@param index of the order to be deleted}
+     */
+    public DeleteOrderCommand(Index index) {
+        requireAllNonNull(index);
+        this.index = index;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        ObservableList<Order> unmodifiableOrderList =
```

Comment:
I think this one fails the law of demeter please open an issue for this with screenshot of this so we will remember to fix this.

Reply:
maybe can just use List<Order> like in person as observable list extends list iirc but I not sure.

Reply:
I think u can just delete this since you no need to find the order to delete as u r just deleting by INDEX